modsettings/ComponentTags.json

```diff
@@ -111,6 +111,34 @@
     {
       "displayed": "DoomCounter_ignore",
       "normalized": "doomcounter_ignore"
+    },
+    {
+      "displayed": "ClassBag_memory_object",
+      "normalized": "classbag_memory_object"
+    },
+    {
+      "displayed": "SeeBag_memory_object",
+      "normalized": "seebag_memory_object"
+    },
+    {
+      "displayed": "SurBag_memory_object",
+      "normalized": "surbag_memory_object"
+    },
+    {
+      "displayed": "NeuBag_memory_object",
+      "normalized": "neubag_memory_object"
+    },
+    {
+      "displayed": "MysBag_memory_object",
+      "normalized": "mysbag_memory_object"
+    },
+    {
+      "displayed": "RogBag_memory_object",
+      "normalized": "rogbag_memory_object"
+    },
+    {
+      "displayed": "aespa_memory_object",
+      "normalized": "aespa_memory_object"
     }
```

Comment:
If there are no memory bags currently "placed", all these tags could be deleted (they will be recreated by the bags when used).
Advantage: These tags would not show up when tagging stuff manually.

Reply:
Let's plan this for a cleanup after 2.4.